common/djangoapps/student/views/dashboard.py

```diff
@@ -60,6 +60,10 @@
 
 log = logging.getLogger("edx.student")
 
+experiments_namespace = WaffleFlagNamespace(name=u'student.experiments')
```

Comment:
Just making sure I understand the goals: The course price and dashboard_metadata will (likely) stay on even after this experiment. But the waffle flag will likely go away once we're sure this performs.

If any of this code is intended to be temporary (like the flag) could we open a ticket to remove it and add a TODO to the code with that ticket number?

Reply:
Yes, that's on my list of things to add today. Gonna write it down and add the TODO comments shortly.